jsa/features/people.feature

```diff
@@ -12,13 +12,13 @@ Feature: People
     And I should see the "People" heading in the "First sidebar" region
     And I should see "This is your Why I Teach block" in the "First sidebar" region
 
-  @content @live
+  @content @deploy
   Scenario: Faculty node
     Given I am on "people/faculty"
     And I click "Jacob Smith" in the "Content Body" region
     Then I should see "Professor of English" in the "Content Body" region
 
-  @content @live
+  @content @deploy
```

Comment:
Oh, I see. I either missed that it was looking for Jacob Smith and Jane Doe in that filter, or you snuck that in to make me look foolish. Clearly the latter.

Yeah, you're right, that probably shouldn't have `@live`.